src/zoid/buttons/component.jsx

```diff
@@ -720,7 +721,32 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                         layout,
                         locale,
                         vault
-                    } }) ? EXPERIENCE.INLINE : '';
+                    } });
+                    
+                    const logger = getLogger();
+
+                    logger
+                        .info('isInlineXOEligible props', { props: JSON.stringify(props) })
+                        .info('isInlineXOEligible eligible', { eligible: String(eligible) })
+                        .track({
+                            [ FPTI_KEY.TRANSITION ]: `inline_xo_eligibility_${ String(eligible) }`
+                        }).flush();
+
+                    if (eligible) {
+                        const inlinexoExperiment = createExperiment('inlinexo', 50, logger);
+                        const treatment = inlinexoExperiment.getTreatment();
+
+                        logger
+                            .info(treatment)
+                            .track({
+                                [FPTI_KEY.EXPERIMENT_NAME]: 'inlinexo',
+                                [FPTI_KEY.TREATMENT_NAME]:  treatment
+                            }).flush();
```

Comment:
I'm not sure that we'd want to potentially flush twice. we might want to remove both flushes above and only call it right before the return statement